backend/dataall/api/Objects/Environment/resolvers.py

```diff
@@ -699,7 +699,51 @@ def get_pivot_role_template(context: Context, source, organizationUri=None):
             )
             raise e
 
+def get_cdk_exec_policy_template(context: Context, source, organizationUri=None):
+    from ....utils import Parameter
 
+    with context.engine.scoped_session() as session:
+        ResourcePolicy.check_user_resource_permission(
+            session=session,
+            username=context.username,
+            groups=context.groups,
+            resource_uri=organizationUri,
+            permission_name=permissions.GET_ORGANIZATION,
```

Comment:
I am not sure about this permission here. Permissions in data.all apply to data.all objects. For example "`DeleteEnvironment` applies to resource: `EnvironmentA`" or "`CreateEnvironment` applies to resource: OrganizationA"

In this case what you have defined is that any user with access to an Organization can download the template. 

I would say that we have 2 options:
1) permissions are not needed as this template is publicly available in GitHub and they do not contain any sensitive info. There is a permission that restricts the creation of environments in the organization
2) use the `LINK_ENVIRONMENT` permission instead. It is more appropiate as this is a pre-step for linking an environment

Reply:
Not applicable, separate PR